susemanager/bin/server-migrator.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+echo
+echo
+echo "==================================================================="
+echo "This script will migrate the Uyuni server from 4.0.1 to 4.0.2 which"
```

Comment:
At least this current migration is "special" as it implies a major OS upgrade. So I'm unsure if we should print this information? Future upgrades might be easier...